src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/CollectionModulesConsistency.fs

```diff
@@ -8,11 +8,17 @@ open NUnit.Framework
 open FsCheck
 open Utils
 
+/// helper function that creates labeled FsCheck properties for equality comparisons
+let consistency name sqs ls arr =
```

Comment:
this is good idea to replace all the assertions with duplicated logic and have better error message.

Should it be `checkConsistency` or even `assertConsistency` (I assume it raises an exception if the equality is not observed).